french/dessine/porte.md

```diff
@@ -3,5 +3,6 @@ La porte qui m'a attire le plus est la porte dessinee avec du miel. Elle brillai
 Donc je decide de
 
 [Manger un peu de miel](miel/manger-du-miel.md)
+[Essayer d'ouvrir la porte faite de chocolat](chocolat/essayer-ouvrir-chocolat.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.